sample/ApiHubTrigger-Batch/function.json

```diff
@@ -0,0 +1,17 @@
+{
+  "bindings": [
+    {
+      "type": "apiHubTrigger",
+      "connectionString": "%dropbox%",
```

Comment:
For these samples, you should define an AzureWebJobsDropBox setting name or something more clear (like the other binding samples) since this will have to be part of our sample setup steps.